### ISSUE TITLE
Missing html lang element

### ISSUE THREAD

**Issue by reynir** [robur-coop/dnsvizor#78]
> There is no html <lang> element that tells screen readers in which language the text is written.

[3.1.1 Language of Page](https://www.w3.org/WAI/WCAG22/Understanding/language-of-page)
